packages/sdk/package.json

```diff
@@ -42,11 +42,9 @@
     "@uma/contracts-frontend": "^0.1.2",
     "@uma/contracts-node": "^0.1.2",
     "axios": "^0.21.4",
-    "bn.js": "^5.2.0",
     "decimal.js": "^10.3.1",
     "highland": "^2.13.5",
-    "lodash-es": "^4.17.21",
-    "web3": "^1.5.3"
+    "lodash-es": "^4.17.21"
```

Comment:
Why do you need this?

Reply:
it wasnt added, the line was just modified, but lodash is used other places in the sdk. i think in multicall libraries